tekton/ci/plumbing-template.yaml

```diff
@@ -69,6 +69,321 @@ spec:
       - name: endtrigger
         resourceRef:
           name: github-check-done
+  - apiVersion: tekton.dev/v1beta1
+    kind: PipelineRun
+    metadata:
+      name: plumbing-ko-build-$(uid)
```

Comment:
We should be able to use the same pipeline for all images, just we different params.
The check-name label is set here, so it should be fine.

Reply:
done :angel: